cobench-examples/dune

```diff
@@ -0,0 +1,7 @@
+(executable
+ (name example)
+ (modules example)
+ (libraries cobench))
+
+(rule
+ (copy ../pipeline/lib/current_bench_json.ml current_bench_json.ml))
```

Comment:
I think you can remove that `rule`, it's only useful within the `cobench` library (but shouldn't be required by the example)